mlrun/runtimes/utils.py

```diff
@@ -31,6 +33,7 @@
 from ..artifacts import TableArtifact
 from ..config import config
 from ..utils import get_in, helpers, logger, verify_field_regex
+from .constants import RunLabels
```

Comment:
Preferably don't use relative imports because they can cause circular import errors

Reply:
Done. fixed the others as well